app/src/main/java/org/simple/clinic/storage/migrations/Migration_76.kt

```diff
@@ -0,0 +1,17 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.simple.clinic.storage.inTransaction
+import javax.inject.Inject
+
+@Suppress("Classname")
+class Migration_76 @Inject constructor() : Migration(76, 77) {
```

Comment:
Can we add a test for this migration and verify that the table was dropped as expected?